automod/engine/persist.go

```diff
@@ -42,6 +43,15 @@ func (eng *Engine) persistAccountModActions(c *AccountContext) error {
 
 	// de-dupe actions
 	newLabels := dedupeLabelActions(c.effects.AccountLabels, c.Account.AccountLabels, c.Account.AccountNegatedLabels)
+	var rmdLabels []string
```

Comment:
embarrassing idiomatic golang question: should this be:

```go
rmdLabels := []string{}
```

specifically b/c it gets `len()` called on it, and passed to `NegateLabelVals` below. probably fine/same either way. "is zero/nil of an array of strings same as empty array"